html/pfappserver/root/src/components/new/BaseInputToggle.vue

```diff
@@ -98,6 +100,14 @@ export const setup = (props, context) => {
     validFeedback
   } = useInputValidator(metaProps, value)
 
+  const toggle = () => {
```

Comment:
This only handles 2-states. It should be able to handle N states (no limit).

Reply:
As we discussed on slack, switch components are just a fancy checkboxes, they are not meant to handle n states. For components requiring more than 2 states we should either use toggle buttons, range sliders, dropdowns, or radio buttons